tasmota/xdrv_60_shift595.ino

```diff
@@ -0,0 +1,114 @@
+/*
+  xdrv_60_shift595.ino - 74x595 shift register family support for Tasmota
+
+  Copyright (C) 2021  Jacek Ziółkowski
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#ifdef USE_SHIFT595
+
+#define XDRV_60           60
+
+const char kShift595Commands[] PROGMEM = "|" D_CMND_SHIFT595_DEVICE_COUNT ;
+void (* const Shift595Command[])(void) PROGMEM = { &CmndShift595Devices };
+
+struct Shift595 {
+  int8_t pinSRCLK;
+  int8_t pinRCLK;
+  int8_t pinSER;
+  int8_t pinOE;
+  int8_t outputs;
+  int8_t first = TasmotaGlobal.devices_present;
```

Comment:
That won't work.
You must set `Shift595.first = TasmotaGlobal.devices_present` in your driver init function